lib/galaxy/datatypes/binary.py

```diff
@@ -169,6 +169,11 @@ def _get_samtools_version( self ):
                 break
         return version
 
+    @staticmethod
+    def merge(split_files, output_file):
+        params = ["samtools", "merge", "-f", output_file, " ".join(split_files)]
```

Comment:
What do you mean exactly with "place the split_files in the params"? Do you mean to change
params = ["samtools", "merge", "-f", output_file, " ".join(split_files)]
by
params = ["samtools", "merge", "-f", output_file, split_files]

and what about shell? do you recommend to leave it as True or False?